hw2/task04/test_hw2_task04.py

```diff
@@ -0,0 +1,12 @@
+from hw2_task04 import cache, function
+
+cache_func = cache(function)
```

Comment:
Please add tests for a function `sum` and argument `[100, 200]`

Reply:
@mirmozavr, please check the comment:

https://github.com/mirmozavr/EPAMHW/pull/9#discussion_r588637141